### PR TITLE
Add timestamp behavior to involved models.

### PR DESCRIPTION
**Pull request by damien-biasotto**
Added timestamp behaviors to involved tables. It should fix issue #644. 

Tables with timestamp behavior:
- Croogo\Blocks\Model\Table\BlocksTable
- Croogo\Blocks\Model\Table\RegionsTable
- Croogo\Comments\Model\Table\CommentsTable
- Croogo\Contacts\Model\Table\ContactsTable
- Croogo\Contacts\Model\Table\MessagesTable
- Croogo\Menus\Model\Table\LinksTable
- Croogo\Menus\Model\Table\MenusTable
- Croogo\Taxonomy\Model\Table\TaxonomiesTable
- Croogo\Taxonomy\Model\Table\TermsTable
- Croogo\Taxonomy\Model\Table\TypesTable
- Croogo\Taxonomy\Model\Table\VocabulariesTable
- Croogo\Users\Model\Table\UsersTable.

I did not rename `updated` field to `modified` field to avoid to mess up with tests.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,7 @@
 language: php
 
+sudo: false
```
Comment (Marlinc) [inline .travis.yml]: Is there a particular reason why you merged this from master?

**Comment by Marlinc** [comment]
Very nice!

File: .travis.yml (hunk 1 above)
Comment (damien-biasotto) [inline .travis.yml]: No sadly it seems that I messed up syncing my fork :/

**Comment by damien-biasotto** [comment]
I'll rebase my PR because there are some commits fro master in it

**Comment by Marlinc** [comment]
The are conflicts since I merged in some stuff from another repository. Could you check that by any chance?

**Comment by damien-biasotto** [comment]
I will check conflicts tonight.

**Comment by Marlinc** [comment]
Thanks a lot!

**Pull request merged**